source/js/buyers-guide/components/creep-vote/creep-vote.scss

```diff
@@ -14,23 +14,6 @@ $btn-shadow-width: 3px;
   }
 }
 
-// #creep-vote targets the <form>
-#creep-vote {
-  &:invalid {
-    button[type="submit"] {
-      @extend .btn.disabled; // we want it to have Bootstrap's .btn.disabled stylings
-
-      color: $gray-20;
-
-      &.btn-secondary:hover {
-        cursor: auto;
-        color: inherit;
-        background: inherit;
-      }
-    }
-  }
-}
-
 .creep-vote {
```

Comment:
Was gonna update line 21 `@extend .btn.disabled;` to `@extend .btn, .disabled;` because
<img width="1036" alt="Screen Shot 2021-01-11 at 5 21 48 PM" src="https://user-images.githubusercontent.com/2896608/104257771-06423680-5433-11eb-8f3c-ab6bbf2d8867.png">
but I realized that that chunk of code is old and should be removed. I think we had it when the form had two required fields.

Reply:
that sounds fine